WordPress/Classes/Extensions/UIImageView+Networking.swift

```diff
@@ -22,9 +29,16 @@ extension UIImageView
             placeholderImage: placeholderImage,
             success: { [weak self]
                 (request: NSURLRequest, response: NSHTTPURLResponse?, image: UIImage) -> Void in
-                
-                self?.image = image
-                success?(image)
+
+                let processedImage: UIImage
```

Comment:
You could just do:

`let processedImage = processor?(image) ?? image`

here? Conciseness vs clarity, perhaps.